collectd-openstack/README.md

```diff
@@ -0,0 +1,300 @@
+
+    [<img src='./img/chart-openstack-neutron-networks.png' width=200px>](./img/chart-openstack-neutron-networks.png)
+
+  - **Routers**: Shows the total number of routers created in all projects.
+
+    [<img src='./img/chart-openstack-neutron-routers.png' width=200px>](./img/chart-openstack-neutron-routers.png)
+
+  - **Subnets**: Shows the total number of subnets created in all projects.
+
+    [<img src='./img/chart-openstack-neutron-subnets.png' width=200px>](./img/chart-openstack-neutron-subnets.png)
+
+  - **Flaoting IPs Available vs Used**: Shows the maximum number of floating IPs available and used in all projects.
+
+    [<img src='./img/chart-openstack-neutron-floating-ips.png' width=200px>](./img/chart-openstack-neutron-floating-ips.png)
+
+  - **Security Groups Available vs Used**: Shows the maximum number of security groups available and used in all projects.
+
+    [<img src='./img/chart-openstack-neutron-security-groups.png' width=200px>](./img/chart-openstack-neutron-security-groups.png)
+
+
+- **INSTANCE**:
+
+  - **VCPUs**: Shows the number of virtual CPUs allocated to the instance.
+
+    [<img src='./img/chart-openstack-instance-vcpus.png' width=200px>](./img/chart-openstack-instance-vcpus.png)
+
+  - **CPU Used %**: Shows the current percentage of CPU usage of an instance.
+
+    [<img src='./img/chart-openstack-instance-cpu-used.png' width=200px>](./img/chart-openstack-instance-cpu-used.png)
+
+  - **CPU %**: Shows the percentage of instance CPU usage over the time.
+
+    [<img src='./img/chart-openstack-instance-cpu.png' width=200px>](./img/chart-openstack-instance-cpu.png)
+
+  - **Memory**: Shows the memory usage of the instance.
+
+    [<img src='./img/chart-openstack-instance-memory.png' width=200px>](./img/chart-openstack-instance-memory.png)
+
+  - **Total Bytes Sent/Received**: Shows the number of bytes sent and received over the network.
+
+    [<img src='./img/chart-openstack-instance-bytes-sent-received.png' width=200px>](./img/chart-openstack-instance-bytes-sent-received.png)
+
+  - **Received vs Transmitted Packers/sec**: Shows the number of network packets sent and received per second.
+
+    [<img src='./img/chart-openstack-instance-packets-sent-received.png' width=200px>](./img/chart-openstack-instance-packets-sent-received.png)
+
+  - **Virtual Disk Read and Write Requests**: Shows the number of virtual disk read and write request.
+
+    [<img src='./img/chart-openstack-instance-vda-read-write-requests.png' width=200px>](./img/chart-openstack-instance-vda-read-write-requests.png)
+
+  - **Size of VDS Reads and Writes**: Shows the size of virtual disk reads and writes.
+
+    [<img src='./img/chart-openstack-instance-vda-read-write-size.png' width=200px>](./img/chart-openstack-instance-vda-read-write-size.png)
+
+  
+A few other details:
+
+* `plugin` is always set to `openstack`
+* `plugin_instance` will contain the project id and name of the project given in the configuration
+
+
+### METRICS
```

Comment:
(this comment applies to line 291) Please check with Akash about the format of this paragraph. I think his script might expect the "For documentation" sentence to be the first, or only, sentence. If you need to change it, the other info could be in a second paragraph.

Reply:
Discussed with Akash, and leaving as it is as script would handle it

Reply:
ok thanks